src/main/java/com/gradle/develocity/bamboo/DevelocityAccessKey.java

```diff
@@ -0,0 +1,82 @@
+package com.gradle.develocity.bamboo;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Objects;
+
+public final class DevelocityAccessKey {
```

Comment:
If I'm reading the code correctly, this class can represent both the "long lived access key" and the new "short lived access token" that is returned from the endpoint Correct?

Maybe we should distinguish between the two?  Even if, perhaps, just the variable names.

Reply:
IMO it's fine to use one class, but it should be renamed to `*AccessCredential`.

Reply:
Yeah, Ryan articulated what I was thinking better.

Reply:
Agree, renamed